tests/integration-tests/tests/scaling/test_scaling/test_scaling_with_manual_actions/pcluster.config.ini

```diff
@@ -0,0 +1,19 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = 5
```

Comment:
Even though this test is only being run for slurm, which we've , it might still be a good idea to parametrize this since you're setting a variable for it anyways in the test.